dashboard/settings.py

```diff
@@ -41,6 +41,10 @@
 
 ALLOWED_HOSTS = os.environ.get("ALLOWED_HOSTS","127.0.0.1,localhost").split(',')
 
+WATCHMAN_TOKEN = getenv('DJANGO_WATCHMAN_TOKEN', None)
```

Comment:
updates needed for the README file or .env.sample file?

Reply:
Oh oops! Forgot about this, I think it's can be any value that you'd just give to access this. By default this is unsecured. But should be in the sample. Thanks!